cloudTopic: /distance_pc
pcTopic: /distance_pc
cloudFrame: /map
cameraImageTopic: /camera_stick/cam0/image
cameraInfoTopic: /camera_stick/cam0/camera_info
cameraFrame: /cam0
outputTopic: /cam0_labels
outputFolder: /home/rene/cla_dataset/
maxDistance: 0.5
overrideOutput: true